manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.yaml

```
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: fjsoft.MyPhoneExplorer
+PackageVersion: 1.8.15
+DefaultLocale: de

```

```suggestion
DefaultLocale: en-US
```
Invalid locale Code.